poc_mobile_verifier/src/scheduler.rs

```diff
@@ -1,7 +1,12 @@
 use crate::{Error, Result};
 use chrono::{DateTime, Duration, Utc};
+use lazy_static::lazy_static;
 use std::ops::Range;
 
+lazy_static! {
+    static ref OFFSET: Duration = Duration::minutes(30);
```

Comment:
would we want to make this a field in the scheduler?

Reply:
Seems like it could make for a good test setup if it’s a configurable value in the verifier settings